vividus-extension-selenium/src/main/java/org/vividus/proxy/Proxy.java

```diff
@@ -112,7 +115,19 @@ public void clearRequestFilters()
     @Override
     public org.openqa.selenium.Proxy createSeleniumProxy()
     {
-        return ClientUtil.createSeleniumProxy(proxyServer, InetAddress.getLoopbackAddress());
+        return Optional.ofNullable(proxyHost)
+                       .map(this::createSeleniumProxy)
+                       .orElseGet(() -> ClientUtil.createSeleniumProxy(proxyServer, InetAddress.getLoopbackAddress()));
+    }
+
+    private org.openqa.selenium.Proxy createSeleniumProxy(String hostName)
```

Comment:
`com.browserup.bup.client.ClientUtil#createSeleniumProxy(com.browserup.bup.BrowserUpProxy, java.net.InetAddress)` ?

Reply:
InetAddress will resolve it, and there is no guarantee that it could be resolved from the machine where the tests running

Reply:
Having host.docker.internal it will throw UnknownHostException with a trace:
{code}
Caused by: java.net.UnknownHostException: host.docker.internal: nodename nor servname provided, or not known
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:930)
	at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1499)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:849)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1489)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1348)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1282)
	at java.base/java.net.InetAddress.getByName(InetAddress.java:1232)
	at org.vividus.proxy.Proxy.createSeleniumProxy(Proxy.java:133)
{code}

Reply:
FYI: https://github.com/browserup/browserup-proxy/pull/318